bootstrap3-typeahead.js

```diff
@@ -61,7 +61,8 @@
     this.followLinkOnSelect = this.options.followLinkOnSelect || this.followLinkOnSelect;
     this.source = this.options.source;
     this.delay = this.options.delay;
-    this.$menu = $(this.options.menu);
+    this.theme = this.option.theme && this.options.themes && this.options.themes[this.options.theme] || Typeahead.defaults.themes[Typeahead.defaults.theme];
+    this.$menu = $(this.options.menu || theme.menu);
```

Comment:
`theme` is also undefined, maybe `this.theme.menu`?

Reply:
You are correct, I'll look into this asap an provide a new PR